File: Tutorials/C#/Tutorials/FormMain.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using TutorialsBase;

namespace TutorialsCS4
{
    public partial class FormMain : TutorialForm
    {
        public FormMain()
        {
            InitializeComponent();
            Text = "NetOffice Tutorials in C#";
            LoadTutorials();
        }

        private void LoadTutorials()
        {
            LoadTutorial(new Tutorial01());
            LoadTutorial(new Tutorial02());
            LoadTutorial(new Tutorial03());
            LoadTutorial(new Tutorial04());
            LoadTutorial(new Tutorial05());
            LoadTutorial(new Tutorial06());
            LoadTutorial(new Tutorial07());
            LoadTutorial(new Tutorial08());
            LoadTutorial(new Tutorial09());
            LoadTutorial(new Tutorial10());
            LoadTutorial(new Tutorial11());
            LoadTutorial(new Tutorial12());
            LoadTutorial(new Tutorial13());
            LoadTutorial(new Tutorial14());
            LoadTutorial(new Tutorial15());
            LoadTutorial(new Tutorial16());
            LoadTutorial(new Tutorial17());
            LoadTutorial(new Tutorial18());
        }
    }
}

File: Tutorials/VB/Tutorials/FormMain.vb
Public Class FormMain

    Public Shared ReadOnly DocumentationBase As String = "https://netoffice.io/documentation/"

    Public Sub New()

        InitializeComponent()
        Me.Text = "NetOffice Tutorials in VB"
        LoadTutorials()

    End Sub

    Private Sub LoadTutorials()

        LoadTutorial(New Tutorial01())
        LoadTutorial(New Tutorial02())
        LoadTutorial(New Tutorial03())
        LoadTutorial(New Tutorial04())
        LoadTutorial(New Tutorial05())
        LoadTutorial(New Tutorial06())
        LoadTutorial(New Tutorial07())
        LoadTutorial(New Tutorial08())
        LoadTutorial(New Tutorial09())
        LoadTutorial(New Tutorial10())
        LoadTutorial(New Tutorial11())
        LoadTutorial(New Tutorial12())
        LoadTutorial(New Tutorial13())
        LoadTutorial(New Tutorial14())
        LoadTutorial(New Tutorial15())
        LoadTutorial(New Tutorial16())
        LoadTutorial(New Tutorial17())

    End Sub

End Class
